### ISSUE TITLE
[FEATURE] Multiple persons per Card

### ISSUE THREAD

**Issue by TMaddox** [Leantime/leantime#267]
**Is your feature request related to a problem? Please describe.**
It isn't possible to add more than one person to a todo card

**Describe the solution you'd like**
Please implement the possibility to add multiple persons to a single card (like in trello)

**Comment by marcelfolaron** [comment]
We believe that each to-do should be owned by one person. If multiple users are working on an assignment they should break up the work into individual to-dos. This increases accountability and allows managers to see who is working on what part.

**Comment by TMaddox** [comment]
Well you are right, if possible we assign one task to one person, that is the case in at least 90% of the time, everything you said about the advantages of one person per card is correct. 

Sometimes we use multiple persons per card. Let me explain how this integrates in our workflow.

- We are building a machine, once it is assembled we got some tests to run, as the tests can only be conducted off site, we got something we call test days, each test day is a card. Of course we need an engineer of different departments on such a test day and we would like to add each of them to the card. Breaking down the test day into smaller pieces wouldn't make any sense from our point of view. (so many small tasks, wouldn't serve a purpose replacing the test day card with 100 small cards explaining all the tasks, which need be done by different people)
- Sometimes we got someone actually doing the task and a second person supervising/consulting, we always added both to the card so they both can keep track of it.
- We have mandatory brainstorming & lessons learned sessions, at the begin of each milestone, which are attended by multiple people

tldr: I agree that one person per todo is the way to go and should be used as much as possible, yet in some scenarios we are really dependant on it. Maybe you could implement it in a way, so that multiple persons per card can be turned on/off per project/company/global settings?

Additional side info: we are currently looking to get away from trello and leantime is our top choice (as it can be self hosted), we all agreed that the implementation of multiple persons per todo is the last step needed to move this project from top choice to perfect :)

**Comment by jasle** [comment]
Sounds to me like there are a little bit different use cases, so maybe a configurable maximum number of persons that can be assigned to an to do would be an option?
Would also like to have the ability to assign more than one person to an to do

**Comment by marcelfolaron** [comment]
I know this issue is rather old but I this topic has come up a few times and I want to close this with an article we have created explaining our reasoning. https://support.leantime.io/support/solutions/articles/154000058088-why-does-leantime-limit-task-assignment-to-one-user-

For now we'll stick to 1 person per ticket.
